packages/devtools_app/test/layout_explorer/layout_explorer_test_utils.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/inspector/diagnostics_node.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'layout_explorer_serialization_delegate.dart';
+
+Future<RemoteDiagnosticsNode> widgetToLayoutExplorerRemoteDiagnosticsNode({
+  @required Widget widget,
+  @required WidgetTester tester,
+  int subtreeDepth = 1,
+}) async {
+  await tester.pumpWidget(
+    MaterialApp(home: Scaffold(body: widget))
+  );
+  final element = find.byWidget(widget).evaluate().first;
+  final nodeJson =
+  element.toDiagnosticsNode(style: DiagnosticsTreeStyle.dense).toJsonMap(
+    LayoutExplorerSerializationDelegate(
+      subtreeDepth: subtreeDepth,
+      service: WidgetInspectorService.instance,
+    ),
+  );
+  return RemoteDiagnosticsNode(nodeJson, null, null, null);
+}
```

Comment:
This is what I meant by my previous comment

Reply:
I like it!